packages/editor/src/components/block-vertical-alignment-toolbar/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+import { Toolbar } from '@wordpress/components';
+import { withViewportMatch } from '@wordpress/viewport';
+import { withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { withBlockEditContext } from '../block-edit/context';
+import { alignTop, alignCenter, alignBottom } from './icons';
+
+const BLOCK_ALIGNMENTS_CONTROLS = {
+	top: {
+		_id: 'top',
```

Comment:
What is the purpose of having this prefixed with underscore `_id` property here? It doesn't look like property which would be part of the public API.

Reply:
Addressed here https://github.com/WordPress/gutenberg/pull/13899/commits/e00a520a735306e57a94d191faf8cf71d02125c7